### PR TITLE
Front: Replace CDN resources with local packages

### PR DESCRIPTION
**Pull request by maximlion**
Replaced Fira-font, bootstrap, summernote, font-awesome & Open-Sans font CDNs with local packages.
I couldn't find a better solution for `summernote` in `xtheme`

https://github.com/shuup/shuup/issues/1404

### CODE REVIEW

File: shuup/xtheme/package.json [hunk 1]
```diff
@@ -18,10 +18,16 @@
     "autoprefixer": "^8.6.4",
     "babel-preset-env": "^1.7.0",
     "bootstrap": "^3.3.6",
-    "less": "^3.0.4",
     "parcel-bundler": "^1.9.7",
+    "shuup-static-build-tools": "file:../../lib/shuup_static_build_tools",
+    "concurrently": "^3.6.0",
+    "jquery": "^3.2.1",
+    "less": "^3.0.4",
+    "parcel-plugin-shuup-maninfest": "file:../../shuup_setup_utils/parcel_plugin_shuup_maninfest",
```
Comment (chessbr) [inline shuup/xtheme/package.json]: remove this.. this is not available anymore

File: shuup/xtheme/static_src/editor/editor.less [hunk 2]
```diff
@@ -185,3 +185,79 @@ form {
     }
     margin-bottom: 5px;
 }
+
+
+@bs-path: "~bootstrap/less";
```
Comment (chessbr) [inline shuup/xtheme/static_src/editor/editor.less]: move all this to a new file: `vendor.less` and import it here

**Comment by maximlion** [comment]
Fixed

File: shuup/front/templates/shuup/front/maintenance.jinja [hunk 3]
```diff
@@ -5,12 +5,12 @@
...
     <link rel="shortcut icon" type="image/x-icon" href="{{ static("img/shuup.ico") }}">
-    <link href='//code.cdn.mozilla.net/fonts/fira.css' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+    <link rel="stylesheet" href="{{ static("shuup/front/css/style.css") }}">
+
     <title>{% trans %}Maintenance mode{% endtrans %}</title>
     <style>
         body {
-            font-family: "Fira Sans", sans-sefif;
+            font-family: "Open Sans", sans-sefif;
```
Comment (chessbr) [inline shuup/front/templates/shuup/front/maintenance.jinja]: do you still need this line? shoundl't this font-family com from `static("shuup/front/css/style.css")` ?

Comment (maximlion) [reply shuup/front/templates/shuup/front/maintenance.jinja]: As I understood it, this is how it's using the imported font from style.css

**Comment by chessbr** [comment]
Replaced by https://github.com/shuup/shuup/pull/1454/

**Pull request closed without merging**